lib/prawn/table/cell/text.rb

```diff
@@ -135,7 +135,13 @@ def text_box(extra_options={})
         #
         def styled_width_of(text)
           options = @text_options.reject { |k| k == :style }
-          with_font { @pdf.width_of(text, options) }
+          if text.empty?
```

Comment:
Yeah, I think `.max || 0` is probably more readable. I'll change that and add a spec.